addons/analytic/models/analytic_mixin.py

```diff
@@ -48,17 +48,21 @@ def _compute_analytic_distribution(self):
         pass
 
     def _search_analytic_distribution(self, operator, value):
-        if operator not in ['=', '!=', 'ilike', 'not ilike'] or not isinstance(value, (str, bool)):
-            raise UserError(_('Operation not supported'))
-        operator_name_search = '=' if operator in ('=', '!=') else 'ilike'
-        account_ids = list(self.env['account.analytic.account']._name_search(name=value, operator=operator_name_search))
+        if operator == 'in' and isinstance(value, (tuple, list)):
+            account_ids = value
+            operator_inselect = 'inselect'
+        else:
+            if operator not in {'=', '!=', 'ilike', 'not ilike'} or not isinstance(value, (str, bool)):
```

Comment:
I prefer the version of 17.0 where don't have the UserError in the middle. It's a bit clearer than in an if/else if (now that it has been merged, but be careful because 17.1 not yet, fw-port are a pain).
I would do it the same way, so we're consistent, and so you can re-remove these double-quotes :smile:  (I don't care about tuple vs set performance on literals, but you can keep sets if you prefer).